tests/acceptance/10_files/14_inform_logging/common.cf.sub

```diff
@@ -0,0 +1,54 @@
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      version => "1.0";
+  # RUNDIRECT is used later to only run the actual test code with cf-agent
+  !RUNDIRECT::
+      bundlesequence  => { default("$(this.promise_filename)") };
+}
+
+#######################################################
+
+bundle agent init
+{
+  files:
+      "$(G.testfile)"
+      delete => init_delete;
+"$(G.testdir)"
+  delete => init_delete;
+}
+
+body delete init_delete
+{
+      dirlinks => "delete";
+      rmdirs   => "true";
+}
+
+#######################################################
+
+bundle agent test
+{
+  commands:
+    "/var/cfengine/bin/cf-agent -D RUNDIRECT -KIf $(testcase.filename) > $(testcase.filename).actual"
+      contain => useshell;
+    "/usr/bin/sed -i 's,$(G.testroot),/tmp,' $(testcase.filename).actual";
+}
+
+body contain useshell
+{
+  useshell => "useshell";
+}
+
+#######################################################
+
+bundle agent check
+{
+  methods:
+    "any" usebundle => dcs_if_diff("$(testcase.filename).actual", "$(testcase.filename).expected", "ok", "not_ok");
+
+  reports:
+    !not_ok::
```

Comment:
negative logic can be insidious. Why not `ok::`?

Reply:
negative outlook? will absolutely change. :+1: